cmd/jaeger/internal/extension/jaegerstaticsampling/README.md

```diff
@@ -0,0 +1,52 @@
+# jaeger_static_sampling
```

Comment:
please add the diagram from https://github.com/jaegertracing/jaeger/pull/5389#issuecomment-2083110099